packages/rocketchat-lib/server/functions/sendMessage.js

```diff
@@ -15,15 +15,20 @@ const objectMaybeIncluding = (types) => {
 	});
 };
 
-const validateAttachmentsFields = attachmentFields => {
-	check(attachmentFields, objectMaybeIncluding({
+const validateAttachmentsFields = attachmentField => {
+	check(attachmentField, objectMaybeIncluding({
 		short: Boolean
 	}));
 
-	check(attachmentFields, objectMaybeIncluding({
+	check(attachmentField, objectMaybeIncluding({
 		title: String,
-		value: String
+		value: Match.OneOf(String, Match.Integer, Boolean)
 	}));
+
+	if (typeof(attachmentField.value) !== undefined) {
```

Comment:
Do we need to enforce it as String? I think it's fine to keep it saved in the original format, what do you think?

Reply:
@rodrigok initially I didn't convert.  But looking at slack's it looks like they are converting to string so added.

But yeah I don't think it matters so much since all accepted will easily be converted to string when displayed in UI

Reply:
I'll ping mobile teams and see which is better compatible

Reply:
@rodrigok confirmed with mobile.  Because both clients are using typed languages having as a string is the best way.